src/main/java/com/edmunds/rest/databricks/restclient/DatabricksRestClientImpl425.java

```diff
@@ -41,18 +43,29 @@
 /**
  * To run on CDH5.7.1, use httpclient4.2.5 version API
```

Comment:
Unrelated to this PR, but, maybe we should remove this comment?

Reply:
Sure, I will modify it to just say that this is an older httpclient implementation.